apps/browser/src/vault/popup/components/vault/view.component.html

```diff
@@ -417,23 +427,52 @@ <h2 class="box-header">
       <!-- Fido2Key -->
       <div *ngIf="cipher.type == cipherType.Fido2Key">
         <div class="box-content-row">
-          <span class="row-label">{{ "keyType" | i18n }}</span>
-          {{ cipher.fido2Key.keyType }}
-        </div>
-
-        <div class="box-content-row">
-          <span class="row-label">{{ "keyCurve" | i18n }}</span>
-          {{ cipher.fido2Key.keyCurve }}
+          <span class="row-label">{{ "username" | i18n }}</span>
+          {{ cipher.fido2Key.userName }}
         </div>
-
         <div class="box-content-row">
-          <span class="row-label">{{ "rpName" | i18n }}</span>
-          {{ cipher.fido2Key.rpName }}
+          <span class="row-label">{{ "typePasskey" | i18n }}</span>
+          {{ "dateCreated" | i18n }} {{ cipher.creationDate | date : "MM/dd/y" }}
         </div>
 
-        <div class="box-content-row">
-          <span class="row-label">{{ "username" | i18n }}</span>
-          {{ cipher.fido2Key.userName }}
+        <div class="box" *ngIf="cipher.fido2Key.canLaunch">
+          <div class="box-content">
+            <div class="box-content-row box-content-row-flex">
+              <div class="row-main">
+                <label
+                  class="row-label draggable"
+                  draggable="true"
+                  (dragstart)="setTextDataOnDrag($event, cipher.fido2Key.launchUri)"
+                  >{{ "application" | i18n }}</label
+                >
+                <span title="{{ cipher.fido2Key.launchUri }}">
+                  <input type="text" [value]="cipher.fido2Key.rpId" readonly aria-readonly="true" />
+                </span>
+              </div>
+              <div class="action-buttons">
+                <button
+                  type="button"
+                  class="row-btn"
+                  appStopClick
+                  (click)="launch(cipher.fido2Key)"
+                  attr.aria-label="{{ 'launch' | i18n }} {{ cipher.fido2Key.launchUri }}"
+                  appA11yTitle="{{ 'launch' | i18n }}"
```

Comment:
suggestion(non-blocking): use input syntax instead of template binding
```suggestion
                  [appA11yTitle]="'launch' | i18n"
```